modules/pxp-module-puppet

```diff
@@ -70,7 +70,8 @@ end
 
 def force_unicode(s)
   # Puppet still returns UTF-8 when locale is C or POSIX, so force encoding to that.
-  if s.encoding == Encoding::ASCII_8BIT
+  if !Puppet.features.microsoft_windows? &&
+    (s.encoding == Encoding::US_ASCII || s.encoding == Encoding::ASCII_8BIT)
```

Comment:
Trying to force C locale on Windows never prompted this issue, so I prefer to target the fix at non-Windows. Targeting it at US-ASCII and ASCII-8BIT also seems safe.